example-mojaloop-backend/values.yaml

```diff
@@ -307,12 +178,12 @@ centralledger-obj:
       -   'password'
     databases:
       -   'mlos'
-    ## @param auth.username DEPRECATED: use `auth.usernames` instead
-    ## @param auth.password DEPRECATED: use `auth.passwords` instead
-    ## @param auth.database DEPRECATED: use `auth.databases` instead
-    username: ""
-    password: ""
-    database: ""
+    # ## @param auth.username DEPRECATED: use `auth.usernames` instead
```

Comment:
I think you have too many #'s here?

Reply:
Yep, fixed.

Reply:
@mdebarros : this too is bloat and not useful so  removed.  ..which of course fixes the too many ##'s issue :-)